completions/_vagrant

```diff
@@ -14,6 +14,13 @@ _vagrant () {
 		fi
 		_describe -t commands "vagrant $1" vms -V1
 	}
+	getboxes() {
+        	local -a boxes
+        	boxes=(
+			$(print -l $(basename ~/.vagrant.d/boxes/*(/)))
```

Comment:
Weird, I have files in `~/.vagrant.d/boxes` that aren't valid box files, empty directories mainly. I can't get `print -l $(basename ~/.vagrant.d/boxes/*(/))` to run by itself though to really test.